src/app/controllers/MenuItemController.ts

```diff
@@ -5,6 +5,10 @@ import { FlashSaleItem } from "./FlashSaleController";
 import { DefaultControllerRequired } from "./Controller";
 import { MutateResult } from "../links/synchronouslinks/GraphQLLink";
 
+export enum FeeType {
```

Comment:
Is there any reason that you don't simply use the enum exported from enums.ts?

Reply:
@junthehacker  The reason I didn't is I'm not sure how the graphql transpiler automatically generates the enums.ts file, I wasn't sure if what I added to Enums was to be deleted or not after it's generated again. I'll go through the PRs and fix the references, and I'll make a ticket for removing the duplicate enums in all controllers so they're all referencing from enums.ts

Reply:
Vili explained it to me, I'll update the ts-sdk after I merge the graphql updates.